widgets/common/src/deploymentsView/table/DeploymentsTable.tsx

```diff
@@ -87,7 +89,11 @@ const DeploymentsTable: FunctionComponent<DeploymentsTableProps> = ({
                     );
                 })}
 
-                {deployments.flatMap(renderDeploymentRow(toolbox, fieldsToShow, selectedDeployment))}
+                {labelsToShow.map(label => (
```

Comment:
To be more precise I think we should have here:
```suggestion
                {labelsToShow.map(key => (
```

Reply:
did not notice that while fixing the other